packages/driver/src/cypress/mocha.ts

```diff
@@ -535,7 +535,7 @@ const patchSuiteAddTest = (specWindow) => {
     const test = args[0]
 
     if (!test.invocationDetails) {
-      test.invocationDetails = $stackUtils.getInvocationDetails(specWindow, $sourceMapUtils.getSourceMapProjectRoot())
+      test.invocationDetails = $stackUtils.getInvocationDetails(specWindow, $sourceMapUtils.getSourceMapProjectRoot(), 'test-body')
```

Comment:
This is probably more accurately described as the `test` since it's not really the body of the test.
```suggestion
      test.invocationDetails = $stackUtils.getInvocationDetails(specWindow, $sourceMapUtils.getSourceMapProjectRoot(), 'test')
```